Formula/xserver.rb

```diff
@@ -0,0 +1,114 @@
+class Xserver < Formula
+  desc "X Window System display server"
+  homepage "https://www.x.org"
+  url "https://gitlab.freedesktop.org/xorg/xserver.git",
+    tag:      "xorg-server-21.1.3",
+    revision: "85397cc2efe8fa73461cd21afe700829b2eca768"
```

Comment:
Any reason why this isn't using tarball https://www.x.org/releases/individual/xserver/xorg-server-21.1.3.tar.xz?

Reply:
The tarball doesn't contain the `hw/xquartz/bundle/chown-bundle.sh`, [build script](https://gitlab.freedesktop.org/xorg/xserver/-/blob/master/hw/xquartz/bundle/meson.build#L63) needs it.
Using `configure` script can avoid this issue but upstream [removed `autotools` support](https://gitlab.freedesktop.org/xorg/xserver/-/commit/c97397dc47f7a25858052f52dd9396ec564065b4).

Reply:
How about the tarball from Gitlab https://gitlab.freedesktop.org/xorg/xserver/-/archive/xorg-server-21.1.3/xserver-xorg-server-21.1.3.tar.bz2, which I am guessing should be generated from repo state.

Unless there is Git modules or git commands directly run, we prefer using tarballs. For example, other formulae that use urls hosted on `gitlab.freedesktop.org` (e.g. `libslirp`, `gst-plugins-rs`, ...).

Reply:
> The tarball doesn't contain the `hw/xquartz/bundle/chown-bundle.sh`, [build script](https://gitlab.freedesktop.org/xorg/xserver/-/blob/master/hw/xquartz/bundle/meson.build#L63) needs it.

This looks like a bug. Can you report it?

Reply:
Issue: https://gitlab.freedesktop.org/xorg/xserver/-/issues/1346